flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.h

```diff
@@ -0,0 +1,44 @@
+/*! @file OIDExternalUserAgentIOSNoSSO.h
+    @copydetails
+        Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Missing attribution/link to the header file it's based of https://github.com/openid/AppAuth-iOS/blob/master/Source/iOS/OIDExternalUserAgentIOS.h

Reply:
@matthewtsmith missed this one :)

Reply:
Oops sorry looks like it's there now. perhaps GitHub showed me a cached copy for some reason